addon/initializers/responsive.js

```diff
@@ -1,17 +0,0 @@
-/**
- * Ember responsive initializer,
- *
- * Injects the media service in all controllers route components and views
- */
-export function initialize() {
-  let application = arguments[1] || arguments[0];
```

Comment:
Do you want to retain this fallback in the new flavor of the initializer just in case? IIRC ember ... 1.12 (?) is when this changed.